data/events.yaml

```diff
@@ -75,6 +75,13 @@
   end_date: March 29 2024
   type: tutorial
 
+- title: "Formalisation of Mathematics: Workshop for Women and Mathematicians of Minority Gender"
+  url: https://www.icms.org.uk/
+  location: International Centre for Mathematical Sciences (ICMS), Edinburgh, United Kingdom
+  start_date: May 27 2024
+  end_date: May 31 2024
+  type: tutorial
```

Comment:
Just to double check: we use the "tutorial" tag for LftCM-style events where novices learn formalization, and "workshop" as a catch-all for most other things. (Maybe we should revisit the tag choices at some point.) Is "tutorial" accurate here?

Reply:
Yes, this is mainly an LftCM-style introductory workshop (although we will also welcome more experienced participants), so I think that the tutorial tag is accurate.